packages/mobile/src/identity/contactMapping.ts

```diff
@@ -36,7 +36,8 @@ export function* doImportContacts() {
   try {
     yield call(getConnectedAccount)
 
-    const result: boolean = yield call(requestContactsPermission)
+    const result: boolean = yield call(checkContactsPermission)
```

Comment:
I would change the name of `results` variable.

Also I think this should be skipped only the first time, because otherwise an user would never be able to sync their contacts after skiping in the NUX lfow afaik